protocol/rest/rest_invoker_test.go

```diff
@@ -1,217 +1,217 @@
 /*
- * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
This header changes caust ci verify failed.
And after you fix it, run `golangci-lint run` locally to make sure it pass, as I see , now your code have linter problems: https://github.com/apache/dubbo-go/runs/6663336503?check_suite_focus=true
@complone